src/electionguard_cli/cli_steps/cli_step_base.py

```diff
@@ -15,17 +15,17 @@ class CliStepBase:
     VERIFICATION_URL_NAME = "verification_url"
 
     def print_header(self, s: str) -> None:
-        click.echo("")
-        click.secho(f"{'-'*40}", fg=self.header_color)
-        click.secho(s, fg=self.header_color)
-        click.secho(f"{'-'*40}", fg=self.header_color)
+        Echo("")
+        Secho(f"{'-'*40}", fg=self.header_color)
```

Comment:
Move header_color and other local members to print utils. 
CliStepBase should not care about the colors of printing to the command line (separation of concerns). It should call print header method or print warning method, and the implementation details of print module should deal with colors.

Reply:
cc @CodingForFun-coder , @jknight777